src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -90,6 +90,7 @@ private void CreateNodeCaches()
             _typeSymbols = new NodeCache<TypeDesc, IEETypeNode>(CreateNecessaryTypeNode);
             _constructedTypeSymbols = new NodeCache<TypeDesc, IEETypeNode>(CreateConstructedTypeNode);
             _methodEntrypoints = new NodeCache<MethodDesc, IMethodNode>(CreateMethodEntrypointNode);
+            _allMethodsOnType = new NodeCache<TypeDesc, AllMethodsOnTypeNode>(CreateAllMethodsOnTypeNode);
```

Comment:
Nit: we don't really need to introduce an abstract method for this. Doing the same thing we do for `_readOnlyDataBlobs` a couple lines below is enough. The abstract methods are leftover from an abstraction that was trying to share NodeFactory with the full AOT compiler. The abstraction didn't work out. There's a TODO comment on top of NodeFactory to get rid of that.

Reply:
Thank you, I've modified it.